_posts/2025-10-22-release-v1.7.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "OpenFE v1.7: SepTop, ABFEs, faster simulations, and more!"
+categories: science update
+---
+
+We are pleased to release version 1.7 of **openfe**!
+
+This release brings along some long-awaited features, notably including new Protocols (SepTop & ABFEs), and performance improvements.
+Please see below for some highlights, you can find more details on our [changelog][].
+
+## New alchemical simulation protocols
+
+### Separated Topologies
+
+We include a new relative binding free energy protocol based on the Separated Topologies (SepTop) method, as defined by [Rocklin et al.][] and [Baumann et al.][]. Using SepTop, you can calculate the difference in binding free energy between two ligands by essentially performing two absolute binding free energy calculations simultaneously in opposite directions. Crucially, this avoids the needs for creating a hybrid topology, making atom mapping unnecessary and allowing for scaffold hopping!
+
+Please see our [SepTop tutorial][] for a demonstration on how to use it! You can also read more about the method in our [SepTop user guide][] entry.
+
+### Absolute Binding Free Energies
+
+We also include a new Absolute Binding Free Energy (ABFE) protocol, based on a partial annhilation scheme. This method allows for the direct calculation of binding affinities by alchemically removing a ligand from its binding site.
+
+Please see our [ABFE tutorial][] and associcated [ABFE user guide entry][] to see how to use the method.
+
+## Performance improvements through better default settings
+
+We have updated our default settings to maximize the performance of `openfe`, leading to a ~ 2x performance improvement compared to the v1.0 release!
+
+The figure below shows the average performance of a single thermodynamic cycle for a pairwise transformation using our relative binding free energy hybrid topology protocol. We see see a drop from ~ 6 hours to ~ 3 hours by switching our default settings and moving to OpenMM 8.2. This can be further improved by layering multiple simulation using the [NVIDIA multi-processing service][]
```

Comment:
```suggestion
The figure below shows the average performance of a single thermodynamic cycle for a pairwise transformation using our relative binding free energy hybrid topology protocol. We see see a drop from ~6 hours to ~3 hours by switching our default settings and moving to OpenMM 8.2. This can be further improved by layering multiple simulation using the [NVIDIA multi-processing service][]
```